builder/dockerfile/instructions/parse.go

```diff
@@ -622,8 +622,8 @@ func errExactlyOneArgument(command string) error {
 	return errors.Errorf("%s requires exactly one argument", command)
 }
 
-func errAtLeastTwoArguments(command string) error {
-	return errors.Errorf("%s requires at least two arguments", command)
+func errNoDestinationArgument(command string) error {
+	return errors.Errorf("%s requires at least two arguments; no destination was given", command)
```

Comment:
Maybe "arguments, but the destination is missing."

Reply:
Perhaps "arguments, but the destination _argument_ is missing." ? (wondering if "destination missing" could be interpreted as the "destination directory is missing"

Reply:
That's fine. I don't know how we can say that there need to be two arguments and the second one is interpreted as the destination. I don't like positional arguments. 🙀

Reply:
LOL, even worse, it can be `src1 src2 src3 dest`. Difficult to capture all that in an error message without dumping an eBook as error message 😂

Reply:
maybe "requires at least two arguments, but only one was provided. Destination could not be determined."